onnxconverter_common/auto_mixed_precision_model_path.py

```diff
@@ -39,6 +39,9 @@ def validate(res1, res2):
 1. keep_io_types=True, so the test_data can be used for both fp32 and fp16 models during the convertion.
 2. about the 'provider', when you want to inference on CPU machine, you must set it to ['CPUExecutionProvider'],
    when you want to do inference on GPU machine, set to ['CUDAExecutionProvider'].
+3. the default value of rtol will be 1e-3 if it is None.
```

Comment:
Just realized all these 5 notes are explaining the parameters of auto_convert_mixed_precision_model_path() function. Could we move them into the comments of this function, so users know its usage specifically? Just like another function [convert_float_to_float16](https://github.com/microsoft/onnxconverter-common/blob/b80f26177a628d0c8be0337705b30fe6b5f2c30b/onnxconverter_common/float16.py#L89).

In addition, we can add comments for other parameters of this function as well.